pom.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.eclipse.jetty</groupId>
+  <artifactId>jetty-docker</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <jetty.version>9.4.29.v20200521</jetty.version>
```

Comment:
This jetty version is different to what is in the dockerfiles.   Perhaps update.sh should update this as well?  Or rather than update finding the lastest, maybe we should take it from the pom?

Reply:
only use for jetty-client in unit test

Reply:
This is for the jetty client in the tests and isn't related to the docker files. We use a jetty HttpClient to do GET requests to verify that the jetty docker containers are working. I could even just do this without the jetty dependency at all which would be easier than updating it with the script.